Header head
int32[] obst_id
float64[] risk